src/pages/AutoDeposit/AutoDepositLandingPage.vue

```diff
@@ -18,6 +18,39 @@
 		</div>
 
 		<!-- Auto Deposit Form -->
+		<div class="auto-deposit-form" v-if="canDisplayForm">
+			<div class="row">
+				<div class="small-12 large-8 columns">
+					<!-- TODO inset form here -->
+				</div>
+			</div>
+		</div>
+
+		<!-- Already Subscribed Notice -->
+		<div class="row" v-if="hasAutoDeposits || hasLegacySubscription">
+			<div class="small-12 large-6 columns">
+				<div class="already-subscribed-msg-wrapper">
+					<h3>
+						You already have an existing auto deposit.
+						Changes can be made in your
+						<a href="/settings/subscriptions">subscription settings</a>.
+					</h3>
+				</div>
+			</div>
+		</div>
+
+		<!-- Monthly Good Notice -->
+		<div class="row" v-if="isMonthlyGoodSubscriber">
+			<div class="small-12 large-6 columns">
+				<div class="already-subscribed-msg-wrapper">
+					<h3>
+						Auto Deposit is not available to Monthly Good subscribers.
```

Comment:
This messaging is new. The previous auto deposit page displays: _"You already have an existing auto deposit. Changes can be made in your subscription settings." When the user has a MG subscription.

Reply:
Good catch